proto/stat_var.proto

```diff
@@ -208,9 +204,11 @@ message SearchStatVarRequest {
 message SearchStatVarResponse {
   // A ranked list of stat vars that aren't included in any of the returned stat
   // var groups.
-  repeated SearchResultEntry stat_vars = 1;
+  repeated EntityInfo stat_vars = 1;
   // A list of stat var group results ranked by relevance.
   repeated SearchResultSVG stat_var_groups = 2;
+  // A list of strings in the names of the results that match the search tokens
```

Comment:
nit: clarify that these are unique strings?

Reply:
done